types/uid2/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for uid2 0.3.3
+// Project: https://github.com/coreh/uid2
+// Definitions by: Levi Bostian <https://github.com/levibostian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function(length: number): string;
```

Comment:
It looks like uid2 also supports an overload with a callback:

https://github.com/coreh/uid2/blob/de1b40992074e01983dd1d145af369eb8bc41137/index.js#L31-L37